pylint/lint/pylinter.py

```diff
@@ -651,12 +651,14 @@ def check(self, files_or_modules: Sequence[str] | str) -> None:
                     self.get_ast, self._iterate_file_descrs(files_or_modules)
                 )
         else:
+            original_sys_path = sys.path[:]
```

Comment:
I'm wondering if we should move the testutil context manager that permit to restore the sys.path at the end into the actual code ?